weave/trace_server/workers/evaluate_model_worker/evaluate_model_worker.py

```diff
@@ -0,0 +1,63 @@
+import asyncio
+from abc import ABC, abstractmethod
+
+from pydantic import BaseModel
+
+from weave.flow.eval import Evaluation
+from weave.scorers.llm_as_a_judge_scorer import LLMAsAJudgeScorer
+from weave.trace.context.weave_client_context import require_weave_client
+from weave.trace.refs import parse_uri
+from weave.trace_server.interface.builtin_object_classes.llm_structured_model import (
+    LLMStructuredCompletionModel,
+)
+
+
+class EvaluateModelArgs(BaseModel):
+    project_id: str
+    evaluation_ref: str
+    model_ref: str
+    wb_user_id: str
+    evaluation_call_id: str
```

Comment:
Is there a reason to differentiate `evaluation_call_id` from `call_id` when the same generated ID is shared at initialization above?

Reply:
honestly just for clarity. this is the expected evaluation call id